portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public final class Task {
+
+  private final String name;
+  private final String text;
+  private final long timestamp;
+
+  public Task(String name, String text, long timestamp) {
+    this.name = name;
+    this.text = text;
+    this.timestamp = timestamp;
+  }
+}
```

Comment:
nit: add new line to get rid of "No newline at end of file" warning

Reply:
Does this mean add a line 29? Or a string with "\n"?

Reply:
i think adding a line 29 should work